handleperson.rb

```diff
@@ -0,0 +1,49 @@
+require './person'
+require './student'
+require './teacher'
+
+class HandlePerson
+  def initialize(arg)
+    @persons = arg
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @persons.push(Teacher.new(age: age, specialization: specialization, name: name))
+    puts('Person added successfully!')
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parents permission? [Y/N]:  '
+    parent_permission = gets.chomp != 'n'
+    @persons.push(Student.new(age: age, classroom: 'Coding Class', name: name, parent_permission: parent_permission))
+    puts('Person added successfully!')
+  end
```

Comment:
- [ ] great job on dividing the code into separate classes 🥇 In order to comply with the DRY method you could put the below part in the `handle_person` method so it will be DRY and still comply with the single responsibility rules:
```
   print 'Age: '
   age = gets.chomp.to_i
   print 'Name: '
   name = gets.chomp
```
- [ ] these 2 methods can be made `private` to make your application more secure.

Reply:
Hey @mikethreels,
Thank you for the review.
We will implement the required changes as soon as possible and request another review.
Cheers 👍